README.md

```diff
@@ -95,6 +95,55 @@ However, here are some rules that we'd like to emphasize since they are not visi
 
 For project ideas relating to the Typelevel ecosystem in the categories of AI/ML, serverless, data streaming, observability, and systems programming, please visit https://typelevel.org/gsoc/ideas/.
 
+### Scala Native / Scala.js Projects
+
+| Title                   | Scala Bazel Rules for Scala.js and Scala Native |
+|-------------------------|------------------------------------------------|
+|Link to Project          | https://github.com/bazelbuild/rules_scala |
+|Brief Description        | This project aims to develop Bazel build rules for Scala.js and Scala Native, enabling efficient and reproducible builds for both platforms. The project will provide first-class support for Scala projects within the Bazel ecosystem, improving integration and developer experience. |
+|Expected Outcome         | Functional and well-documented Bazel rules that allow compiling, testing, and packaging Scala.js and Scala Native projects. Demonstration projects showcasing usage. |
+|Prerequisites            | Experience with Scala, build tools (SBT, Bazel), and Scala.js/Scala Native basics. Some familiarity with Bazel rule definitions is a plus. |
+|Expected Difficulty      | Medium |
+|Expected Time Commitment | Large project - 350 hours |
+|Mentor                   | Wojciech Mazur (GitHub: [@WojciechMazur](https://github.com/Wojciech Mazur), Email: [wmazur@virtuslab.com](wmazur@virtuslab.com))|
+|Co-mentor                | TODO |
+
+| Title                   | JMH-Compliant Benchmarking Framework for Scala Native & Scala.js |
+|-------------------------|-------------------------------------------------------------|
+|Link to Project          | [Scala Native](https://scala-native.org/) / [Scala.js](https://www.scala-js.org/) |
+|Brief Description        | This project aims to implement a benchmarking framework similar to JMH (Java Microbenchmark Harness)  allowing for accurate and reliable performance measurements on non JVM platforms. Both Scala Native and Scala.js cannot consume a modified JVM bytecode emitted by JMH framework. The goal is to create a runtime implementation for executing microbenchmarks and a Scala compiler plugin performing required transformations of Scala.js / Scala Native code based on JMH framework compiletime annotations. |
```

Comment:
The alternative can be designing a pure Scala library instead of integrating with JHM. The initial idea for implementation was due to the fact that JHM is de facto a standard and mostly the only choice right now.